docs/API.md

```diff
@@ -619,8 +624,8 @@ An object with the following properties:
 * `"ipv4"`: A boolean, optional. (default `true`). Used to configure the test and enable IPv6 tests.
 * `"nameservers"`: A list of *name server* objects, optional. (default: `[]`). Used to perform un-delegated test.
 * `"ds_info"`: A list of *DS info* objects, optional. (default: `[]`). Used to perform un-delegated test.
-* `"profile"`: A *profile name*, optional. (default `"default"`). Run the tests using the given profile.
-* `"config"`: **Deprecated**. A string, optional. Ignored. Specify `"profile"` instead.
+* `"profile"`: A *profile name*, optional. (default `"default"`). Run the tests using the given profile. It must be a profile listed in the [available profiles].
+* `"config"`: **Deprecated**. Will cause an errror, specify `"profile"` instead.
```

Comment:
I think `config` is enough deprecated and could be removed.

Reply:
Done.